### PR TITLE
Css redo - UI updates to sidenav and pageheader

### PR DESCRIPTION
**Pull request by danchong**
First set of updates to test across apps.

Updates include:
- adjust pageheader opacity, box shadow, height
- sidenav general UI styling, first pass of many
- fix pageheader overlapping scrollbar issue

### CODE REVIEW

**Comment by Nedda** [comment]
Hey!
So I noticed a new Design System page. Did we want to use this as a display of all our base styles, and experiment with them? Should we make this part of this PR, or can we make it part of a different one?

I noticed there was some in-line code highlighting as well (the default stuff). I have a separate branch that I'm working on for visual-stack docs, and I imported a syntax highlighter. Wanted to confirm where we go with this before I make too many changes.

Other than that I'm gonna look into why travis is failing for your PR.

**Comment by danchong** [comment]
Thank you Nedda, I might have found the issue which had to do with the linting. I was calling in a PageHeaderSection but it wasn't being used. Removed it and getting a clean build. Should I resubmit a new PR?

**Comment by Nedda** [comment]
@danchong go ahead a push the fixes, lets see if it helps with Travis. 
If it passes Travis, we can get to merging it in :)

File: packages/visual-stack/src/css/redo.css [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+@font-face {
+    font-family: 'gibson-light';
+    src: url('../fonts/gibson/gibson-light-webfont.woff2') format('woff2'),
+         url('../fonts/gibson/gibson-light-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+
+/*! --- START CJ Bootstrap Overrides --- */
```
Comment (Nedda) [inline packages/visual-stack/src/css/redo.css]: Had a question here. Are we still using bootstrap css, or is this a generic term we're using for to mean CJ's stylings. I was under the impression we're not using any actual bootstrap stuff, and wanted to remove it from the code. However I don't know what exactly is going on with this css-redo and don't want to nuke anything we're actually using!

**Pull request merged**